view/statistics-flow-roles.js

```diff
@@ -173,10 +173,10 @@ exports['statistics-main'] = function () {
 
 		section({ class: 'pad-if-pagination' }, pagination),
 		br(),
-		section({ class: "section-primary statistics-table-scrollable" },
 			data.map(function (result) {
-				var mode = modes.get(location.query.mode || 'daily');
-				return table({ class: 'statistics-table' },
+			var mode = modes.get(location.query.mode || 'daily');
+			return div({ class: 'overflow-x width-1050px' },
```

Comment:
responsive container may not work here... still that should not happen:

![layout_problem](https://cloud.githubusercontent.com/assets/7392217/25737598/045f7c6e-3179-11e7-8dca-0d5315eb92f6.png)